interfaces/vglass.xml

```diff
@@ -0,0 +1,56 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<!-- This file defines the interfaces and methods. -->
+<node>
+    <interface name="mil.af.secureview.vglass">
+        <method name="show_text">
+            <arg name="on" type="b" direction="in"/>
+            <arg name="text" type="s" direction="in"/>
+            <!-- annotation name="org.freedesktop.DBus.Method.NoReply" value="true"/ -->
+        </method>
+        <method name="clear_text">
+            <!-- annotation name="org.freedesktop.DBus.Method.NoReply" value="true"/ -->
+        </method>
+
+        <signal name="render_complete">
```

Comment:
I don't think this is hooked up, and it would be excessive it if were.

Reply:
It is connected: https://gitlab.com/vglass/glass/-/blob/master/renderer/common/renderer.cpp#L41
and emitted: https://gitlab.com/vglass/glass/-/blob/master/renderer/common/renderer.cpp#L561
I am not quite sure to what end.

Reply:
I ran dbus-monitor and didn't see it. vGlass dbus proxy defined it as a signal, but it should be a slot?  Unclear if some magic is generating, or should generate, the signal sending.

Reply:
I saw 'render_complete' after display hotplug & reconfiguration.  So it seems to be mis-named.